src/Sylius/Bundle/CoreBundle/Mailer/ShipmentEmailManagerInterface.php

```diff
@@ -11,11 +11,13 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\CoreBundle\EmailManager;
+namespace Sylius\Bundle\CoreBundle\Mailer;
 
 use Sylius\Component\Core\Model\ShipmentInterface;
 
 interface ShipmentEmailManagerInterface
 {
     public function resendConfirmationEmail(ShipmentInterface $shipment): void;
+
+    public function sendConfirmationEmail(ShipmentInterface $shipment): void;
```

Comment:
```suggestion
    public function sendConfirmationEmail(ShipmentInterface $shipment): void;

    public function resendConfirmationEmail(ShipmentInterface $shipment): void;
```
IMO seems more natural